src/frequenz/sdk/timeseries/_formula_engine/_formula_generators/_pv_power_formula.py

```diff
@@ -66,3 +67,39 @@ def generate(self) -> FormulaEngine:
             builder.push_clipper(0.0, None)
 
         return builder.build()
+
+    def _get_pv_meters(self) -> abc.Set[int] | None:
+        component_graph = connection_manager.get().component_graph
+
+        pv_inverters = list(
+            comp
+            for comp in component_graph.components()
+            if comp.category == ComponentCategory.INVERTER
+            and comp.type == InverterType.SOLAR
+        )
+        if not pv_inverters:
+            return None
+
+        pv_meters: set[int] = set()
```

Comment:
I wonder if it'd be better to return this empty `set` rather than `None` when there is no PV inverters. If this comment applies, changes in the function definition will be needed.

Reply:
true,  fixed.